packages/webdriverio/tests/findStrategy.test.js

```diff
@@ -31,6 +31,12 @@ describe('selector strategies helper', () => {
         expect(element.value).toBe('search.input')
     })
 
+    it('should find an element using "name" method with a . in the name', () => {
+        const element = findStrategy('[name="searchinput[@isDisplayed=\'true\']"]')
```

Comment:
I don't see `.` symbol in the selector that you put

Reply:
Sorry, the description should be different.